wpeverest/everest-forms
# Tweak - Remove smart tags icon from default value option for Number field.

### All Submissions:






### Changes proposed in this Pull Request:
This PR removes the smart tags inserting icon from default value option in Number field.

### How to test the changes in this Pull Request:

1. Create a form
2. Add Number field
3. See if the smart tags icon appears or not for the default value option. It should not be present.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Remove smart tags icon from default value option for Number field.
